src/plugins/dashboard/public/application/dashboard_app_controller.tsx

```diff
@@ -527,9 +526,6 @@ export class DashboardAppController {
       filterManager.getFilters()
     );
 
-    timefilter.disableTimeRangeSelector();
-    timefilter.disableAutoRefreshSelector();
```

Comment:
Just adding a comment here about our conversation that this removal really doesn't have anything to do with the rest of this PR, just that it's unnecessary code that doesn't do anything.